src/components/Icons/_Icons.scss

```diff
@@ -28,24 +30,84 @@
     margin-left: var(--space-xxs);
   }
 
+  // SIZE
```

Comment:
This just came in from UX.  The values in the size classes need to be changed to the following:
- small: `0.875rem` (14px)
- medium: `1.125rem` (18px)
- large: `--space-m` 1.5rem (24 px)
- xlarge: `--space-l` 2rem 
- xxlarge: `--space-xl` 3rem 
- xxxlarge: `--space-xxl` 4rem 

Continue ot use the CSS vars as much as possible and just use the straight `rem` values for `small` and `medium`.

Reply:
Gotch. Any explanation for this update?

Reply:
I am writing an Iconography section for the Style Guide and I am working with Kathy to resolve descrepencies between the Figma files and how the component was executed.